aidial_adapter_vertexai/dial_api/resource.py

```diff
@@ -0,0 +1,181 @@
+import base64
+import mimetypes
+from abc import ABC, abstractmethod
+from typing import List
+
+from aidial_sdk.chat_completion import Attachment
+from pydantic import BaseModel, Field, root_validator, validator
+
+from aidial_adapter_vertexai.dial_api.storage import FileStorage, download_file
+from aidial_adapter_vertexai.utils.resource import Resource
+from aidial_adapter_vertexai.utils.text import truncate_string
+
+
+class ValidationError(Exception):
+    message: str
+
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(message)
+
+
+class MissingContentType(ValidationError):
```

Comment:
Let's be consistent in naming
```
ValidationError
MissingContentTypeError
UnsupportedContentTypeError
```

Reply:
Ah, and I missed the same thing in Bedrock PR 😞

Reply:
Fixed